wagtail/admin/templates/wagtailadmin/shared/chooser_breadcrumb.html

```diff
@@ -1,24 +1,29 @@
 {% load i18n wagtailadmin_tags %}
 
-<ul class="breadcrumb">
-    {% if show_locale_labels %}
-        <li class="breadcrumb-item">
-            {% if locale_options %}
-                {% if selected_locale %}
-                    {% include "wagtailadmin/shared/locale_selector.html" with translations=locale_options locale=selected_locale only %}
-                {% else %}
-                    {% include "wagtailadmin/shared/locale_selector.html" with translations=locale_options locale=page.locale only %}
-                {% endif %}
+{% if show_locale_labels %}
+    <li>
```

Comment:
remove this wrapping `li` as this content is no longer in a list so it does not need it. 

Maybe wrap in a `<div class="locale-options">`

Remember that li must be within a `ul` and now that we have moved it out we must change the dom element - https://www.w3.org/WAI/tutorials/page-structure/content/